<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
SPDX-FileCopyrightText: none
SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-KDE-Accepted-GPL
 </author>
 <class>ClipDurationDialog_UI</class>
 <widget class="QDialog" name="ClipDurationDialog_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="end_label">
     <property name="text">
      <string>Crop end:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="TimecodeDisplay" name="m_cropStart" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="TimecodeDisplay" name="m_pos" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="position_label">
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>218</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="TimecodeDisplay" name="m_cropEnd" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="crop_label">
     <property name="text">
      <string>Crop start:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="TimecodeDisplay" name="m_dur" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="duration_label">
     <property name="text">
      <string>Duration:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="KMessageWidget" name="mode_info">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="closeButtonVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimecodeDisplay</class>
   <extends>QWidget</extends>
   <header>widgets/timecodedisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ClipDurationDialog_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClipDurationDialog_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
